<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DressShopWindow</class>
 <widget class="QMainWindow" name="DressShopWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="QWidget" name="store_tab">
       <attribute name="title">
        <string>库存</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="store_tool_bar" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>64</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="horizontalSpacing">
              <number>4</number>
             </property>
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QPushButton" name="btn_add_new_goods">
               <property name="minimumSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
               <property name="text">
                <string>新增
商品</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="store_main_view">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QDockWidget" name="dockWidget_4">
               <property name="features">
                <set>QDockWidget::DockWidgetMovable</set>
               </property>
               <property name="allowedAreas">
                <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
               </property>
               <property name="windowTitle">
                <string>总览</string>
               </property>
               <widget class="QWidget" name="dockWidgetContents_4">
                <layout class="QGridLayout" name="gridLayout_4">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QTableView" name="goods_view"/>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QDockWidget" name="dockWidget_2">
               <property name="features">
                <set>QDockWidget::DockWidgetMovable</set>
               </property>
               <property name="allowedAreas">
                <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
               </property>
               <property name="windowTitle">
                <string>详情</string>
               </property>
               <widget class="QWidget" name="dockWidgetContents_2">
                <layout class="QGridLayout" name="gridLayout_5">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QTableView" name="inventory_view"/>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="orders_tab">
       <attribute name="title">
        <string>订单</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="application_menu">
    <property name="title">
     <string>application</string>
    </property>
    <property name="icon">
     <iconset resource="../res/resource.qrc">
      <normaloff>:/icons/baseline_home_black_36dp.png</normaloff>:/icons/baseline_home_black_36dp.png</iconset>
    </property>
   </widget>
   <widget class="QMenu" name="undo_menu">
    <property name="title">
     <string>undo</string>
    </property>
    <property name="icon">
     <iconset resource="../res/resource.qrc">
      <normaloff>:/icons/baseline_undo_black_36dp.png</normaloff>:/icons/baseline_undo_black_36dp.png</iconset>
    </property>
   </widget>
   <widget class="QMenu" name="redo_menu">
    <property name="title">
     <string>redo</string>
    </property>
    <property name="icon">
     <iconset resource="../res/resource.qrc">
      <normaloff>:/icons/baseline_redo_black_36dp.png</normaloff>:/icons/baseline_redo_black_36dp.png</iconset>
    </property>
   </widget>
   <addaction name="application_menu"/>
   <addaction name="undo_menu"/>
   <addaction name="redo_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../res/resource.qrc"/>
 </resources>
 <connections/>
</ui>
